app/hooks/use-ledger.ts

```diff
@@ -11,8 +11,8 @@ export enum LedgerConnectStep {
   HasAddress,
 }
 
-const SAFE_ASSUME_REAL_DEVICE_DISCONNECT_TIME = 1000;
-const POLL_LEDGER_INTERVAL = 750;
+const SAFE_ASSUME_REAL_DEVICE_DISCONNECT_TIME = 1_000;
```

Comment:
What was broken and why does this fix it? Seems sus

Reply:
Need to ping ledger to detect disconnections. If it's too quick, > 1 req is made on the "Preview screen" which crashes Nano Xes.  Partly their bug too

Reply:
>_> wow okay.. lol nice ledger